### ISSUE TITLE
Re-order resource types

### ISSUE THREAD

**Issue by DanRStevens** [OutpostUniverse/OPHD#985]
Reference:
https://github.com/OutpostUniverse/OPHD/pull/983#discussion_r654735980

```cpp
ResourceNamesRefined[0],
ResourceNamesRefined[2],
ResourceNamesRefined[1],
ResourceNamesRefined[3],
```

The index order `0`, `2`, `1`, `3`, is strange, confusing, and unexpected. I believe I've noticed this in a few places now.

If we typically need to use resources out of natural index order, perhaps we should re-order what the indexes mean?

**Comment by ldicker83** [comment]
I would agree if it's a typical use case. I've been using it as Common Metals/Common Minerals/Rare Metals/Rare Minerals but display may show Common/Rare Metals, Common/Rare Minerals.

I'm not opposed to changing it if that would make more sense.
